--- old/tiled/io/xml/XMLMapTransformer.java	2009-09-27 00:37:03.000000000 +0200
+++ src/tiled/io/xml/XMLMapTransformer.java	2009-09-27 00:37:28.000000000 +0200
@@ -759,7 +759,7 @@
             builder.setEntityResolver(entityResolver);
             InputSource insrc = new InputSource(in);
             insrc.setSystemId(xmlPath);
-            insrc.setEncoding("UTF8");
+            insrc.setEncoding("UTF-8");
             doc = builder.parse(insrc);
         } catch (SAXException e) {
             e.printStackTrace();
